harshita/level1/letters.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+    char letter;
+    cout << "Enter a letter: ";
+    cin >> letter;
+    if(letter>=65&&letter<=90){
```

Comment:
That's correct !! But `letter >= 'A' && letter <= 'Z'` is more readable code.

Reply:
oh okay here I have use ASCII values, next time will use it.